com.unity.render-pipelines.high-definition/Runtime/Material/Water/Water.hlsl

```diff
@@ -127,15 +113,141 @@ BSDFData ConvertSurfaceDataToBSDFData(uint2 positionSS, SurfaceData surfaceData)
     bsdfData.perceptualRoughness = PerceptualSmoothnessToPerceptualRoughness(surfaceData.perceptualSmoothness);
     bsdfData.roughness = PerceptualRoughnessToRoughness(bsdfData.perceptualRoughness);
 
-    bsdfData.foamColor = surfaceData.foamColor;
+    bsdfData.foam = surfaceData.foam;
     bsdfData.refractionColor = surfaceData.refractionColor;
     bsdfData.tipThickness = surfaceData.tipThickness;
 
-    bsdfData.specularSelfOcclusion = surfaceData.specularSelfOcclusion;
-
     return bsdfData;
 }
 
+//-----------------------------------------------------------------------------
+// conversion function for deferred
+
+// Layout of the water gbuffer
+// GBuffer0 R16G16B16A16 (64 bits)
```

Comment:
Note: Have in mind that what matter is the bandwith, more than the number of buffer, so here with 3 64bit it is equivalent to 6 RGBA height + depth buffer. Let's move forward with it for now, but it is near to impractical on xboxone

Reply:
There is still room for reducing the foot print of each pixel, this is just a temporary state